impeller/renderer/backend/metal/compute_pass_mtl.mm

```diff
@@ -258,8 +258,10 @@ static bool Bind(ComputePassBindingsCache& pass,
 
     // Special case for linear processing.
     if (height == 1) {
-      int64_t threadGroups =
-          std::max(width / maxTotalThreadsPerThreadgroup, 1LL);
+      int64_t threadGroups = std::max(
```

Comment:
Forgot these were ints so we were capped at 1.

Reply:
Can you add a test for this?

Reply:
Yes. Doing so required me to update the vulkan compute shaders to provide a specialization constant which contains the actual wg size, as these need to agree with the value in compute_pass.vk for anything to work correctly.